libs/application/templates/estate/src/lib/dataSchema.ts

```diff
@@ -274,15 +274,16 @@ export const estateSchema = z.object({
   stocks: z
     .object({
       organization: z.string(),
-      nationalId: z.string(),
+      nationalId: z.string().optional(),
```

Comment:
Won't  this produce an equivalent type `string | undefined`? If so, the below code would fail on undefined as  `kennitala.isValid()` accepts only `string | number`.

Reply:
Oh, ignore that. I see in the code below the additional check for undefined `nationalId && kennitala.isValid(nationalId)`